app/src/main/kotlin/com/wire/android/di/CoreLogicModule.kt

```diff
@@ -148,6 +148,11 @@ class UseCaseModule {
     fun provideObserveSyncStateUseCase(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId) =
         coreLogic.getSessionScope(currentAccount).observeSyncState
 
+    @ViewModelScoped
+    @Provides
+    fun provideSetConnectionPolicyUseCase(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId) =
```

Comment:
is this provider used somewhere?

Reply:
Aaah yeah, I forgot to take this one out.

I had a different implementation earlier that I ended up not using. I'll take it out.